### PR TITLE
Mes 790 add independent driving label + MES 791 & MES792 Show me questions  

### PR DESCRIPTION
**Pull request by zielinskikamil**
* I moved independent driving label to post-test summary
* Imporove a bit layout (changed divs to ios-rows & ion cols)
* Extracted to constant.ts tell me and show me questions
* rename tell me question modal

screen shot 2018-05-10 at 13 39 37
screen shot 2018-05-10 at 13 39 19

### CODE REVIEW

File: src/app/constants.ts [hunk 1]
```diff
@@ -14,3 +14,200 @@ export const DEFAULT_LANG = 'en';
 export const SYS_OPTIONS = {
   systemLanguage: DEFAULT_LANG
 };
+
+export interface IQuestionOption {
+  id: string;
+  question: string;
+  keyWords: string;
+  today: string;
+  last7days: string;
```
Comment (danielchildsbjss) [inline src/app/constants.ts]: these can be numbers?

File: src/components/questions-modal/questions-modal.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
 </ion-header>
 
 <ion-content padding>
-  <ion-row *ngFor="let question of tellMeQuestions; let i = index" class="mes-text-md align-items-center mes-bottom-border">
+  <ion-row *ngFor="let question of options; let i = index" class="mes-text-md align-items-center mes-bottom-border">
```
Comment (mathieuhendey) [inline src/components/questions-modal/questions-modal.html]: Is the `let i = index` needed?

File: src/components/questions-modal/questions-modal.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { ViewController, NavParams } from 'ionic-angular';
+
+interface IQuestionOption {
```
Comment (danielchildsbjss) [inline src/components/questions-modal/questions-modal.ts]: Don't we already have this interface in another file?

File: src/pages/post-test-summary/post-test-summary.ts [hunk 4]
```diff
@@ -83,4 +111,13 @@ export class PostTestSummaryPage {
     });
     prompt.present();
   }
+
+  private mapSafetyQuestion(showMeQuestion?): string[] {
```
Comment (mathieuhendey) [inline src/pages/post-test-summary/post-test-summary.ts]: Is the parameter really optional? Doesn't look like it from the function implementation

File: src/components/questions-modal/questions-modal.html (hunk 2 above)
Comment (zielinskikamil) [reply src/components/questions-modal/questions-modal.html]: is needed, it is padded to toggle option few lines below

File: src/pages/post-test-summary/post-test-summary.ts (hunk 4 above)
Comment (zielinskikamil) [reply src/pages/post-test-summary/post-test-summary.ts]: good catch. It was at the beginning but recently it has changed

Comment (zielinskikamil) [reply src/pages/post-test-summary/post-test-summary.ts]: ooo, sec. in constructor this method is executed without param (as at this stage we dont really know what was the show me quesiton)

Comment (mathieuhendey) [reply src/pages/post-test-summary/post-test-summary.ts]: Ah okay

**Pull request merged**